openmoc/plotter.py

```diff
@@ -78,12 +78,12 @@ def plot_tracks(track_generator):
   num_azim = track_generator.getNumAzim()
   spacing = track_generator.getTrackSpacing()
   num_tracks = track_generator.getNumTracks()
-  coords = track_generator.retrieveTrackCoords(num_tracks*4)
+  coords = track_generator.retrieveTrackCoords(num_tracks*6)
```

Comment:
I added constants to `constants.h` as follows:

`#define NUM_VALUES_PER_RETRIEVED_TRACK 6`
`#define NUM_VALUES_PER_RETRIEVED_SEGMENT 7`

These values are used in the `plotter` and `TrackGenerator` in setting the length of the coords array used to retrieve the tracks and segments. In the future, if developers want to bundle more info in the arrays they just need to change this value in one location and add/extract the necessary data to the array. I'm open to other suggestions if you see a better method.